global-setup.js

```diff
@@ -0,0 +1,5 @@
+// Set time zone to UTC for jest config
```

Comment:
Why is this needed? Also this config will be overriden by real TZ out of local testing right?

Reply:
This config is for jest unit test.
I have added the screenshot for manual testing.